tidy3d/components/heat_charge/simulation.py

```diff
@@ -436,7 +591,7 @@ def plot_boundaries(
         plane = Box(center=center, size=size)
 
         # get boundary conditions in the plane
-        boundaries = self._construct_heat_boundaries(
+        boundaries = self._construct_heat_charge_boundaries(
             structures=structures,
             plane=plane,
             boundary_spec=self.boundary_spec,
```

Comment:
will this plot heat and condution bc at the same time? That's probably will not be very useful as they will overlap with each other generally. Probably we should add a parameter in `plot_boundaries` which tells whether to plot heat or charge bcs, so that when `plot_property` is called, only corresponding bcs are plotted

Reply:
I'm thinking of changing all the features plot based on `property`  so that if `property="heat_conductivity"`, sources, bcs and monitors will be plotted for the HEAT setup, whereas it `property="electric_conductivity"` then all features will be CONDUCTION.